src/withdrawals/main.eas

```diff
@@ -66,6 +65,12 @@
   iszero                ;; [calldatasize != 0]
   jumpi @check_input
 
+  ;; Reject any callvalue here to prevent lost funds.
```

Comment:
we shouldn’t do this as the callvalue is used to cover the requests fee further down the line

Reply:
Ah, I see!